code/modules/power/supermatter/supermatter.dm

```diff
@@ -488,13 +534,15 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			supermatter_anomaly_gen(src, FLUX_ANOMALY, rand(5, 10))
 		if(power > SEVERE_POWER_PENALTY_THRESHOLD && prob(5) || prob(1))
 			supermatter_anomaly_gen(src, GRAVITATIONAL_ANOMALY, rand(5, 10))
-		if(power > SEVERE_POWER_PENALTY_THRESHOLD && prob(2) || prob(0.3) && power > POWER_PENALTY_THRESHOLD)
+		if((power > SEVERE_POWER_PENALTY_THRESHOLD && prob(2)) || (prob(0.3) && power > POWER_PENALTY_THRESHOLD))
```

Comment:
this does clarify logic but order of operations would have evaluated this properly before
![image](https://user-images.githubusercontent.com/10467687/70864931-f99c9780-1f2d-11ea-94ce-63b17b0f9ac2.png)

Reply:
That's good, I'm glad to know it wasn't causing any issues.

Reply:
people should get into the habit of using the brackets anyway

Reply:
yeah i didnt mean to discourage it i just wanted to clarify that it didn't cause any bug which is what they said *might* be the case in their original pr sorry for the confusion folks